dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/preheat/supplier/EnrollmentSupplier.java

```diff
@@ -73,4 +70,13 @@ public void preheatAdd(TrackerObjects trackerObjects, TrackerPreheat preheat) {
           });
     }
   }
+
+  private List<Enrollment> getEnrollmentsByProgram(List<Program> programs) {
+    TypedQuery<Enrollment> query =
+        entityManager.createQuery(
+            "FROM Enrollment e WHERE e.deleted = false AND e.program IN (:programIds)",
```

Comment:
Can we settle on using lowercase in our SQL 😋 it's more comfortable to type and does not look like screaming 😂